beacon-chain/rpc/eth/beacon/BUILD.bazel

```diff
@@ -17,7 +17,7 @@ go_library(
         "validator.go",
     ],
     importpath = "github.com/prysmaticlabs/prysm/v4/beacon-chain/rpc/eth/beacon",
-    visibility = ["//beacon-chain:__subpackages__"],
+    visibility = ["//visibility:public"],
```

Comment:
should all these files be completely public? or should we move these structs to some sub folder